public/js/apps/management/app.jsx

```diff
@@ -1,20 +1,33 @@
 import 'shims';
 
-import React, { Component } from 'react';
+import React, { Component, PropTypes } from 'react';
 import { Provider, Connector } from 'redux/react';
 import { bindActionCreators } from 'redux';
 
 import reduxConfig from 'redux-config';
 import * as managementActions from 'actions/management';
 import * as userActions from 'actions/user';
+import { parseQuery } from 'utils';
 
 import ModalError from 'views/modal-error';
-import Management from 'views/management';
-import ManageCards from 'views/manage-cards';
+import { default as DefaultManagement } from 'views/management';
+import { default as DefaultManageCards } from 'views/manage-cards';
 
 
 export default class ManagementApp extends Component {
 
+  static propTypes = {
+    ManageCards: PropTypes.object,
```

Comment:
I didn't get around to checking if this should be node, but I suspect not since this is a class not a function returned by createClass.